src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,26 @@
+const State          = require('../../_common/storage').State;
+const Client         = require('../../app/base/client');
+const BinarySocket   = require('../../app/base/socket');
+const getElementById = require('../../../javascript/_common/common_functions').getElementById;
+
+const ResponsibleTrading = (() => {
+
+    const onLoad = () => new Promise(resolve => {
+        BinarySocket.wait('landing_company').then(() => {
+            const current_landing_company_shortcode = State.getResponse('authorize.landing_company_name') || 'default';
+            const residence = Client.get('residence');
```

Comment:
we should fallback to IP address country found in `website_status` so we can handle logged out clients as well

Reply:
`State.getResponse('website_status.clients_country')`

Reply:
You are right. Fixed.